dask/dataframe/io/parquet/fastparquet.py

```diff
@@ -27,28 +32,45 @@
 from .utils import Engine
 
 
-def _paths_to_cats(paths, scheme):
-    """Extract out fields and labels from directory names"""
-    # can be factored out in fastparquet
+def _paths_to_cats(paths, file_scheme):
+    """
+    Extract categorical fields and labels from hive- or drill-style paths.
```

Comment:
I think my preference is to just bump the fastparquet requirement to 0.3.2. Do you have a preference @martindurant?

Reply:
Sounds OK to me to update the requirement.

Reply:
In practice, @ig248, I suppose this would involve a runtime check at the start of `read_parquet` for the fastparquet engine. Raise an ImportError if it's too old. How does that sound to you? Is this too aggressive of an update for users?

Reply:
@TomAugspurger part of the issue is that it needs `fastparquet>0.3.2`, as of yet *unreleased*. I am all for a required version, but my versions are locked so nothing would break unexpectedly :D

Reply:
Ah, I thought it was `>=0.3.2`. Yes that's a bit aggressive then :)

I can try to make a fastparquet release today. In the meantime, this is probably OK to include until we require 0.3.3+.

Reply:
That would be grand and would save us merging a PR just to change it again in a few days! Need to check that fastparquet provides a `__version__`